documentation/docs/15-configuration.md

```diff
@@ -165,6 +166,7 @@ When pages are prerendered, the CSP header is added via a `<meta http-equiv>` ta
 
 Environment variable configuration:
 
+- `devModes` — a list of Vite modes to be considered "development" modes. When the app is run in any of these modes, `$env/dynamic/*` will be populated in a less-efficient manner compatible with ecosystem tools such as Vitest and `vite-node`.
```

Comment:
Maybe it is my lack of knowledge on Vite and how to configure a development mode. I tried to understand what this says. I could extend the SvelteKit project configuration by adding `foobar` to the `devModes` config. How would I make my project then run in `foobar` mode? Is this something that we should include in the docs as well? Or is that just the lack of the developer (hey, me) reading that documentation?